scripts/push-to-assets-branch

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+set -e
+set -x
+
+rm -rf /tmp/tmp-assets
+mkdir -p /tmp/tmp-assets
+cp -r assets/ index.yaml index.html sha256sum/ /tmp/tmp-assets
+if [[ -d charts ]]; then
+  cp -r charts/ /tmp/tmp-assets
+fi
+git checkout .
+git clean -fd
+git fetch ${1} ${2}
```

Comment:
We should add a check on the top of this file to ensure that these were passed in before making any changes and store these into readable variables, e.g.

```
if [[ -z ${1} || -z ${2} ]]; then
    echo "Usage: ./scripts/push-to-assets-branch <package-branch> <assets-branch>"
    exit 1
fi
package_branch=$1
asset_branch=$2
```

Reply:
good catch, added